{
    "fixtures": ["server"],
    "auth": "auth_functions:invalid_auth",
    "vars": {
        "api_base": "http://localhost:5000"
    },
    "stages": [
        {
            "name": "test_with_invalid_auth",
            "request": {
                "url": "{{ api_base }}/api/protected",
                "method": "GET"
            },
            "response": {
                "verify": {
                    "status": 200
                }
            }
        }
    ]
}